docs/modules/ROOT/pages/how-tos/configuring/activation-keys-subscription.adoc

```diff
@@ -147,45 +150,70 @@ ubi-9-baseos-rpms
 ubi-9-codeready-builder                                                                                                           Red Hat Universal Base Image 9 (RPMs) - CodeReady Builder`
 ----
 
-In the example above, the source RPM repositories are not enabled for
+In the example above, the source RPM repositories are not enabled for the following repositories:
+
 ----
 ubi-9-appstream-rpms                                                                                                           
 ubi-9-baseos-rpms                                                                                                               
 ubi-9-codeready-builder    
 ----
 
-You must locate the source repos in redhat.repo and change `+ENABLED = 0+` to `+ENABLED = 1+`.
+You must locate and enable the appropriate RPM repositories in `+redhat.repo+` by changing `+enabled = 0+` to `+enabled = 1+`.
```

Comment:
Is this about more than just the source repositories? Does it need to be done before configuring the secret or can the enabled repositories be configured afterwards?

Technically, we only need to enable the source repositories if we want to be able to create the source containers with that specific task.